enterprise-ws-spec/src/main/asciidoc/Overview.adoc

```diff
@@ -301,22 +301,22 @@ in one of two different ways:
 . A Stateless Session Bean: The service provider implements the Web
 service business logic by creating a stateless session Bean that
 implements the methods of the Service Endpoint Interface as described in
-the Enterprise JavaBeans 3.0 specification.
+the Jakarta Enterprise Beans 4.0 specification.
 . A Java class: The service provider implements the Web service business
-logic according to the requirements defined by the JAX-RPC or JAX-WS
+logic according to the requirements defined by the Jakarta XML Web Services
 Servlet based service implementation model.
-. A Singleton Session Bean: The service provider implements the JAX-WS
-Web service business logic by creating a singleton session bean that
-implements the methods of the Service Endpoint Interface as described in
-the EJB 3.1 specification.
+. A Singleton Session Bean: The service provider implements the Jakarta XML
+Web Services Web service business logic by creating a singleton session bean
+that implements the methods of the Service Endpoint Interface as described in
+the Jakarta Enterprise Bean 4.0 specification.
```

Comment:
I'd perhaps remove version number for the Jakarta Enterprise Bean spec, it's not used in the reference to servlet spec, so why should it be there for EJB?